### PR TITLE
Slight revisions on about page copy. Enables "About" page in production.

### PR DESCRIPTION
**Pull request by SeanKennyNF**
Part of #231. Just some slight copy revisions from the copy document for this page. Nothing too serious :)

I'm also turning the about page on in production in this PR so that we have less to remember to do when we start linking people over to this dashboard.

### CODE REVIEW

File: app/about/about-our-data/page.tsx [hunk 1]
```diff
@@ -15,24 +15,37 @@ export default function DataExtractionPage() {
...
+        <Link className="inline text-link" href="https://www.who.int/" target="__blank" rel="noopener noreferrer">World Health Organization</Link>
+        <p className="inline"> to visualize and synthesize results from the </p>
         <Link className="inline text-link" href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/technical-guidance/early-investigations" target="__blank" rel="noopener noreferrer">Unity seroprevalence studies</Link>
         <p className="inline">. SeroTracker and the World Health Organization are grateful for German Federal Ministry of Health (BMG) COVID-19 Research and Development funding to support this effort. </p>
       </div>
       <div className="mb-4">
         <Link className="inline text-link" href="https://www.mapbox.com/" target="__blank" rel="noopener noreferrer">Mapbox </Link>
         <p className='mb-4 inline'>supports SeroTracker&apos;s mapping infrastructure.</p>
       </div>
+      <h2 className={headerClassname}>New dashboard: ArboTracker </h2>
```
Comment (SeanKennyNF) [inline app/about/about-our-data/page.tsx:36]: Header changed and section moved up from it's old location.

File: app/about/about-our-data/page.tsx [hunk 2]
```diff
@@ -15,24 +15,37 @@ export default function DataExtractionPage() {
...
+      <div className="mb-4">
+        <p className='inline'> ArboTracker data was collected through a systematic review of published arbovirus seroprevalence studies in collaboration with partners at the </p>
+        <Link className='inline text-link' href='https://coloradosph.cuanschutz.edu/research-and-practice/centers-programs/globalhealth/research-projects/arbovirus-research-consortium' target="__blank" rel="noopener noreferrer">Arbovirus Research Consortium</Link>
```
Comment (SeanKennyNF) [inline app/about/about-our-data/page.tsx:39]: Just one quick link we wanted changed.

File: app/about/about-our-data/page.tsx [hunk 3]
```diff
@@ -45,20 +58,11 @@ export default function DataExtractionPage() {
       </div>
       <div className="mb-4">
         <p className='inline'>To make us aware of new arbovirus seroprevalence studies or arbovirus studies that we have not yet captured, please fill out </p>
-        <Link className='inline text-link' href='https://forms.gle/pKNiMiMYr6hiKnXx8'>this form.</Link>
-        <p className='inline'>For SARS-CoV-2 submissions, please use </p>
-        <Link className='inline text-link' href='https://docs.google.com/forms/d/e/1FAIpQLSdvNJReektutfMT-5bOTjfnvaY_pMAy8mImpQBAW-3v7_B2Bg/viewform'>this form</Link>
+        <Link className='inline text-link' href='https://forms.gle/pKNiMiMYr6hiKnXx8' target="__blank" rel="noopener noreferrer">this form.</Link>
+        <p className='inline'> For SARS-CoV-2 submissions, please use </p>
```
Comment (SeanKennyNF) [inline app/about/about-our-data/page.tsx:62]: Whoops

File: app/about/layout.tsx [hunk 4]
```diff
@@ -31,10 +31,6 @@ export default async function AboutPageBaseLayout (props: AboutPageBaseLayoutPro
 
   const dehydratedState = dehydrate(queryClient);
 
-  if (process.env.NEXT_PUBLIC_ABOUT_PAGE_ENABLED !== "true") {
-    return notFound();
-  }
```
Comment (SeanKennyNF) [inline app/about/layout.tsx:36]: I removed `NEXT_PUBLIC_ABOUT_PAGE_ENABLED` and just turned it on in production. I figured we're okay with this given that the copy is has now been looked over by Mairead and Harriet and isn't my placeholder copy anymore and that we're going to start linking people here pretty soon.

**Review by HimanshuRanka — APPROVED** [review]

**Pull request merged**